db/migrate/20160602153918_increase_decimal_precision.rb

```diff
@@ -0,0 +1,11 @@
+class IncreaseDecimalPrecision < ActiveRecord::Migration
+  def up
+    change_column :price_policies, :overage_rate, :decimal, precision: 12, scale: 4
+    change_column :price_policies, :reservation_rate, :decimal, precision: 12, scale: 4
```

Comment:
Cool. This increase in precision makes perfect. Seems like it should have been this before.